signer/src/transaction_signer.rs

```diff
@@ -440,14 +437,33 @@ where
     async fn is_valid_stackstransaction_sign_request(
```

Comment:
I'd probably call this `assert_valid_...` or something since you're not returning a bool anymore. The `is_valid..(..).await?;` above gave me a little "wha?" :)

Reply:
Ooops, forgot to get around to doing that.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/545/commits/2773454d18c6c8a8776c9485d2e751aefffd6a60.